### PR TITLE
go-fastly support for kafka endpoint sasl options

### PR DESCRIPTION
**Pull request by kellymclaughlin**
Add the following fields in order to support SASL:

* parse_log_keyvals
* auth_method
* request_max_bytes
* user
* password

### CODE REVIEW

**Review by mccurdyc — APPROVED** [review]
This LGTM. I checked out the branch and ran the tests

File: fastly/kafka.go [hunk 1]
```diff
@@ -29,6 +29,11 @@ type Kafka struct {
 	CreatedAt         *time.Time `mapstructure:"created_at"`
 	UpdatedAt         *time.Time `mapstructure:"updated_at"`
 	DeletedAt         *time.Time `mapstructure:"deleted_at"`
+	ParseLogKeyvals   bool       `mapstructure:"parse_log_keyvals"`
+	RequestMaxBytes   uint       `mapstructure:"request_max_bytes"`
+	AuthMethod        string     `mapstructure:"auth_method"`
+	User              string     `mapstructure:"user"`
+	Password          string     `mapstructure:"password"`
```
Comment (phamann) [inline fastly/kafka.go:36]: Minor nit, the project convention is to keep the metadata fields `CreatedAt` etc as the last in the struct.

Comment (kellymclaughlin) [reply fastly/kafka.go:33]: Thanks, I fixed that up.

**Comment by kellymclaughlin** [comment]
@mccurdyc or @phamann I don't have permissions to merge this. If it looks good would one of you mind merging it for me? Thanks!

**Pull request merged**